query/src/main/java/io/keen/client/java/KeenQueryConstants.java

```diff
@@ -3,10 +3,10 @@
 /**
  * Commonly used constants in the HTTP query request
  *
- * @author claireyoung
+ * @author claireyoung, baumatron, masojus
  * @since 1.0.0, 05/18/15
  */
-public class KeenQueryConstants {
+class KeenQueryConstants {
```

Comment:
I'm confused about this one, though. How could any client code be accessing these constants if they're all default access, which for class types is package-private, unless they wrote their code to be in this package, which is pretty suspect to begin with?